### ISSUE TITLE
Update to Spark 2.1.1

### ISSUE THREAD

**Issue by seufagner** [DmitryBe/clickhouse-spark-connector#1]
Do you think about update to support Spark 2.1.1 and clickhouse-jdbc 0.1.25 ?

**Comment by fabiorhs** [comment]
I agree and, since time passed, I'd say update to Spark 2.4.0 and clickhouse-jdbc 0.1.50.
Thank you.
